docs/sphinx/source/modelchain.rst

```diff
@@ -374,13 +399,17 @@ vs. DataFrame)!
 Inferring models
 ~~~~~~~~~~~~~~~~
 
-How does ModelChain infer the appropriate model types? ModelChain uses a
-series of methods (ModelChain.infer_dc_model, ModelChain.infer_ac_model,
-etc.) that examine the user-supplied PVSystem object. The inference
-methods use set logic to assign one of the model-specific methods, such
-as ModelChain.sapm or ModelChain.snlinverter, to the universal method
-names ModelChain.dc_model and ModelChain.ac_model. A few examples are
-shown below.
+When ModelChain's attributes are not assigned when the instance is created,
+ModelChain can infer the appropriate model from data stored on the ``PVSystem``
+object. ModelChain uses a set of methods (e.g., :py:meth:`~pvlib.modelchain.ModelChain.infer_dc_model`,
+:py:meth:`~pvlib.modelchain.ModelChain.infer_ac_model`, etc.) that examine the
+parameters on the user-supplied PVSystem object. The inference methods use set
+logic to assign one of the model-specific methods, such as
+:py:meth:`~pvlib.modelchain.ModelChain.sapm` or :py:meth:`~pvlib.modelchain.ModelChain.snlinverter`,
+to the universal method names ``ModelChain.dc_model`` and ``ModelChain.ac_model``,
+respectively. A few examples are shown below. Inferrence methods generally work
```

Comment:
```suggestion
respectively. A few examples are shown below. Inference methods generally work
```

Reply:
this suggestion might have been buried/overlooked.